client/src/services/userService.js

```diff
@@ -1,6 +1,6 @@
 import http from "./httpService";
 
-const apiEndpoint = "http:localhost:5000/api/user";
+const apiEndpoint = "http://localhost:5000/api/user";
```

Comment:
This can't be just /api/user? Or perhaps you can make the domain name be an environment variable of some sort?